### PR TITLE
Federation editorial edits

### PR DESCRIPTION
**Pull request by paul-grassi**

### CODE REVIEW

File: sp800-63c/sec3_definitions.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ A communication channel that uses approved encryption where the initiator of the
 The process of establishing confidence in the identity of users or information systems.
 
 #### Authentication Protocol
-A defined sequence of messages between a claimant and a verifier that demonstrates that the claimant has possession and control of one or more valid authenticators to establish his/her identity. Secure authentication protocols also demonstrate to the claimant that he or she is communicating with the intended verifier.
+A defined sequence of messages between a claimant and a verifier that demonstrates that the claimant has possession and control of one or more valid authenticators to establish his/her identity. Secure authentication protocols also demonstrate to the claimant that they are communicating with the intended verifier.
```
Comment (jricher) [inline sp800-63c/sec3_definitions.md]: missed his/her -> their

File: sp800-63c/sec5_assertions.md [hunk 2]
```diff
@@ -80,13 +80,17 @@ Assertions SHALL use audience restriction techniques to allow an RP to recognize
 
 #### <a name="ppi"></a> 5.2.5. Pairwise Pseudonymous Identifiers
 
-In some circumstances, it is desirable to prevent the subscriber's account at the IdP from being easily linked at multiple RPs through use of a common identifier. When using pairwise pseudonymous subject identifiers within the assertions generated by the IdP for the RP, the IdP SHALL generate a different identifier for each RP as described in [Section 5.2.6](#ppi-gen) below.
+In some circumstances, it is desirable to prevent the subscriber's account at the IdP from being easily linked at multiple RPs through use of a common identifier. 
+
+#### <a name="ppi"></a> 5.2.5.1. General Requirements
+
+When using pairwise pseudonymous subject identifiers within the assertions generated by the IdP for the RP, the IdP SHALL generate a different identifier for each RP as described in [Section 5.2.6](#ppi-gen) below.
```
Comment (jricher) [inline sp800-63c/sec5_assertions.md]: Link label should be changed to 5.2.5.2

**Review by jricher — CHANGES_REQUESTED** [review]

**Comment by paul-grassi** [comment]
Fixed in latest commit.

**Comment by paul-grassi** [comment]
@jricher I also kept the PPI href pinned to 5.2.5 rather than moving it to 5.2.5.1.

**Review by jricher — APPROVED** [review]

**Pull request merged**